deployers.nix

```diff
@@ -0,0 +1,125 @@
+let
```

Comment:
ideally, I'd prefer the `deployers.nix` file under a `deployments` directory to be coherent with cardano-ops / jormungandr-ops